src/main/java/me/totalfreedom/totalfreedommod/discord/commands/UptimeCommand.java

```diff
@@ -0,0 +1,79 @@
+package me.totalfreedom.totalfreedommod.discord.commands;
+
+import me.totalfreedom.totalfreedommod.discord.command.DiscordCommandImpl;
+import me.totalfreedom.totalfreedommod.util.FLog;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.MessageBuilder;
+import net.dv8tion.jda.api.entities.Member;
+
+import java.awt.*;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Collections;
+import java.util.List;
+
+public class UptimeCommand extends DiscordCommandImpl
+{
+    @Override
+    public String getCommandName()
+    {
+        return "uptime";
+    }
+
+    @Override
+    public String getDescription()
+    {
+        return "Returns the uptime of the VPS.";
```

Comment:
Perhaps a little specific? Maybe "Returns the uptime of the server" or something else because not necessarily everyone uses a VPS